apps/docs/src/app/platform/component-docs/platform-display-list-item/platform-display-list-item-examples/platform-display-list-item-border-less-example.component.ts

```diff
@@ -13,7 +13,7 @@ export interface Movie {
 })
 export class PlatformDisplayListItemBorderLessExampleComponent {
     items: Movie[] = [
-        { title: 'Star War', secondary: 'Next session', navigationIndicator: true, link: '/platform/home' },
+        { title: 'Star War Item looooooooooooooooooooooooooooonoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong title', secondary: 'Next session Item looooooooooooooooooooooooooooonoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong message', navigationIndicator: true, link: '/platform/home' },
```

Comment:
I'd add `Lorem ipsum` text instead of this

Reply:
Hi @InnaAtanasova , Updated with Lorem ipsum text. Please do have a look.